### DISCUSSION TITLE
Blackhole just randomly stops working after a couple of minutes

### DISCUSSION THREAD

**Discussion by TheNerd18** [ExistentialAudio/BlackHole#646] [Q&A] [answered]
Hey, I'm currently setting up a livestream and running a 32 channel desk into logic and outputting it into 2 channels of the blackhole. I have an aggregate device set up which take the 32 channels from the desk, 16 blackhole channels and 2 channels for external headphones. After routing the audio correctly through logic, it then runs into OBS with the Audio Input being the blackhole 16ch. This works perfectly fine for the first couple of minutes but then it just completly cuts out without me even touching anything. Some help would be really appreciated. Thanks

(Below are pictures of the aggregate device setup and the logic routing
![Screenshot 2022-08-10 at 3 48 24 PM](https://user-images.githubusercontent.com/49015943/183808820-e9d0c4cf-ef17-4911-97e6-940d1699c34f.png)
![Screenshot 2022-08-10 at 3 44 17 PM](https://user-images.githubusercontent.com/49015943/183808831-42f29329-5d85-471b-8bad-3bb305a2c6b0.png)
)

**Comment by devinroth** [comment, marked as answer]
https://github.com/ExistentialAudio/BlackHole#why-is-audio-glitching-after-x-minutes-when-using-a-multi-output-or-an-aggregate

**Reply by TheNerd18** [reply]
Awesome thank you so much. Very silly mistake by me haha
